samples/analyze.js

```diff
@@ -285,18 +285,32 @@ async function analyzeVideoTranscription(gcsUri) {
   const [operation] = await client.annotateVideo(request);
   console.log('Waiting for operation to complete...');
   const [operationResult] = await operation.promise();
```

Comment:
I believe we want an inner `async function` which would be contained with the region tags, as well as the outer async function.

Otherwise the code between the region tags isn't copy-pastable.

Reply:
I will make that change